x/bundles/keeper/logic_end_block_handle_upload_timeout_test.go

```diff
@@ -661,7 +824,12 @@ var _ = Describe("logic_end_block_handle_upload_timeout.go", Ordered, func() {
 		// ASSERT
 		bundleProposal, _ = s.App().BundlesKeeper.GetBundleProposal(s.Ctx(), 0)
 		Expect(bundleProposal.NextUploader).To(Equal(i.STAKER_0))
-		Expect(bundleProposal.StorageId).To(Equal("P9edn0bjEfMU_lecFDIPLvGO2v2ltpFNUMWp5kgPddg"))
+		Expect(bundleProposal.StorageId).To(Equal(""))
+
+		// check if previous bundle got finalized
+		finalizedBundle, _ := s.App().BundlesKeeper.GetFinalizedBundle(s.Ctx(), 0, 0)
+		Expect(finalizedBundle.Uploader).To(Equal(i.STAKER_0))
+		Expect(finalizedBundle.StorageId).To(Equal("y62A3tfbSNcNYDGoL-eXwzyV-Zc9Q0OVtDvR1biJmNI"))
 
 		// check if next uploader got not removed from pool
```

Comment:
Shouldn't the next uploader (Staker 2) be removed now?

Reply:
Staker 2 got removed from the pool, it is assterted in line 840

Reply:
He is removed. But it's not so clear in the test.
I updated the test.